src/main/java/core/basesyntax/operation/ReturnHandler.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.service.ShopService;
+import core.basesyntax.transaction.FruitTransaction;
+
+public class ReturnHandler implements OperationHandler {
+    private ShopService shopService;
+
+    public ReturnHandler(ShopService shopService) {
+        this.shopService = shopService;
+    }

```

Comment:
It's a code quality issue. Avoid using service classes in handlers. Each handler should contain only the logic for the operation it handles. The service class should use these handlers instead.